File: Python/stronghold/rna.py
# Transcribing DNA into RNA
import sys


def transcribe_dna(dna):
    return ''.join('U' if nucleotide == 'T' else nucleotide for nucleotide in dna)


def main():
    reader = sys.stdin
    dna = next(reader).strip()
    result = transcribe_dna(dna)
    print(result)


if __name__ == '__main__':
    main()

File: OCaml/src/stronghold/rna.ml
let transcribe_dna (dna : string) : string =
    String.map (fun nucleotide -> if nucleotide = 'T' then 'U' else nucleotide) dna


let () =
    let dna = read_line () in
    let result = transcribe_dna dna in
    print_endline result
